camel/src/main/java/com/github/theprez/manzan/configuration/DestinationConfig.java

```diff
@@ -122,7 +129,7 @@ private Map<String, String> getUriAndHeaderParameters(final String _name, Sectio
         List<String> exclusions = new LinkedList<>(Arrays.asList(_exclusions));
         exclusions.addAll(Arrays.asList("type", "filter", "format"));
         for (final String sectionKey : sectionObj.keySet()) {
-            if (exclusions.contains(sectionKey)) {
+            if (exclusions.contains(sectionKey) || sectionKey.startsWith("componentOptions.")) {
```

Comment:
Can we make this `componentOptions.` prefix a static variable that we can reference from Config.java instead of hardcoding?

Reply:
Yessir